src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -272,6 +272,9 @@ class WorkItemListPage {
   get filterDropdown () {
     return  element(by.id("wi_filter_dropdown"));
   }
+  filterDropdownId () {
```

Comment:
This code is correct - but - why do we need 2 functions that access the same UI element?